### DISCUSSION TITLE
Record elapesed time

### DISCUSSION THREAD

**Discussion by damulaoshi** [neperfepx/neper#171] [Q&A] [answered]
Dear Romain,
Is there any way that I can generate a file (for example ".logtime") to record elapsed time for neper -M module? (for neper -T module,  I know maybe I can use "-morphooptilogtime total") 
Is there any easy command in neper I can use to record elapsed time ? 

Thank you before and wish to hear your answer.
Best regards,
Damu

**Comment by rquey** [comment, marked as answer]
You can use Neper's terminal output, which includes the elapsed time. Here are a simple way using shell commands:

```
$ neper -T -n 10000 | grep Elapsed | awk '{print $5}'
```

PS. `tee` is also an interesting command.

**Reply by damulaoshi** [reply]
Thank you so much for your answer! Yes I have tried, "tee" command is very useful for me. 

Thank you so much!
